### DISCUSSION TITLE
v1.2.1

### DISCUSSION THREAD

**Discussion by snowby666** [snowby666/poe-api-wrapper#30] [Announcements]
Update example.py + minor changes
**Full Changelog:** [`v1.2.0...v1.2.1`](https://github.com/snowby666/poe-api-wrapper/compare/v1.2.0...v1.2.1)

<hr /><em>This discussion was created from the release <a href='https://github.com/snowby666/poe-api-wrapper/releases/tag/v1.2.1'>v1.2.1</a>.</em>

**Comment by johnd0e** [comment]
Great release! Thank you!

The only issue is with `!history` command in example, again.
ATM it is paginated now. BUT! It does query all items at once, and starts to show anything only after the whole request gets completed.
And history api is not fast!

For example, if i call `!history 1`, I have to wait 55 seconds until I am able to see any output (`You are on page 1 of 235`).

**Comment by johnd0e** [comment]
Another (rather minor, but weird) issue in example.
On bot selection screen, if I press Ctrl-C - I get `Your choice: Invalid cookie. Please try again.`

**Comment by johnd0e** [comment]
And I have one wish for example.
On bot selection screen numeration starts from `0`.
For the sake of consistency, I would like to have same with thread selection screen: `[<] Return to Bot selection` + `[0] Create a new Thread`
